crates/virtio-bindings/src/virtio_net/mod.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Unikie
+// SPDX-License-Identifier: BSD-3-Clause OR Apache-2.0
+
+mod generated;
+pub use generated::*;
+
+use std::fmt::{Debug, Formatter, Result};
+use std::mem::{size_of, transmute};
+
+impl Debug for virtio_net_hdr_v1 {
+    fn fmt(&self, f: &mut Formatter) -> Result {
+        // SAFETY: As of Linux v6.0, all union fields have compatible types.
+        // https://doc.rust-lang.org/reference/items/unions.html#reading-and-writing-union-fields
+        let csum = unsafe { self.__bindgen_anon_1.csum };
```

Comment:
This will not really print correct values on big endian platforms, but I am not sure how many people would actually run this on big endian to begin with.

Reply:
That's also the case for all the generated Debug implementations too, right?  I could probably fix it here, but I think being inconsistent with all the other types would probably be even more confusing.